### ISSUE TITLE
jSSC EXCEPTION_ACCESS_VIOLATION on Windows 10 x64

### ISSUE THREAD

**Issue by nsxfreddy** [jeffpiazza/derbynet#147]
I'm hitting an EXCEPTION_ACCESS_VIOLATION running derby-timer.jar on Windows 10 x64. I've gotten it with both the Oracle runtime and the OpenJDK runtime (via AdoptOpenJDK).

```
C:\derbynet>java.exe -jar derby-timer.jar -d SmartLineDevice -logdir C:\derbynet\TimerLogs -trigger-file-directory C:\derbynet\TriggerFiles -lanes 4 -p <password> https://<server>/derbynet
Starting log file C:\derbynet\TimerLogs\timer-20210101-1411.log
COM3
   Sending HELLO
   Sending HELLO
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x000000007110b5db, pid=18624, tid=0x0000000000000438
#
# JRE version: OpenJDK Runtime Environment (8.0_275-b01) (build 1.8.0_275-b01)
# Java VM: OpenJDK 64-Bit Server VM (25.275-b01 mixed mode windows-amd64 compressed oops)
# Problematic frame:
# C  [jSSC-2.8_x86_64.dll+0xb5db]
#
# Failed to write core dump. Minidumps are not enabled by default on client versions of Windows
#
# An error report file with more information is saved as:
# C:\derbynet\hs_err_pid18624.log
#
# If you would like to submit a bug report, please visit:
#   https://github.com/AdoptOpenJDK/openjdk-support/issues
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
```

A bit of digging suggests this could be a known issue with the original jSSC that has been fixed by a fork? [This issue](https://bugs.openjdk.java.net/browse/JDK-8203772?focusedCommentId=14181807&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14181807) links to a defunct issue filed against jSSC, but [this issue](https://github.com/java-native/jssc/issues/65) discusses it on the java-native/jssc fork. Would it be worth looking at switching to the fork of jSSC that is maintained?

I had no issue with the current derby-timer.jar on Linux (Raspberry Pi).

**Comment by jeffpiazza** [comment]
Updating jssc seems to fix this.  Build 1189 (just now posted) incorporates the change.

**Comment by nsxfreddy** [comment]
Build 1190 fixed it (1189 had the issue in #148). Thanks!
